tests/Unit/Helpers/DataStructures/VectorImplTestHelper.hpp

```diff
@@ -500,22 +496,79 @@ struct VectorOrArrayAt {
 // given an explicit template parameter pack `Wraps`, wrap the elements of
 // `operand`, passed by pointer, element-by-element, and return the
 // resulting tuple of wrapped elements.
-template <typename... Wraps, typename... Operands, size_t... Is>
+template <typename WrapTuple, typename... Operands, size_t... Is>
 auto wrap_tuple(std::tuple<Operands...>& operand_values,
+                const WrapTuple& wraps,
                 std::index_sequence<Is...> /*meta*/) noexcept {
-  return std::make_tuple(wrap(Wraps{}, get<Is>(operand_values))...);
+  return std::make_tuple(wrap(get<Is>(wraps), get<Is>(operand_values))...);
 }
 
+// CHECK forwarding for parameter pack expansion. Return value also required for
+// easy parameter pack use.
+template <typename Approx>
+SPECTRE_ALWAYS_INLINE int call_check_approx(double a, double b,
```

Comment:
`const` on at least the `double`s. Probably `const &` on the approx

Reply:
can't `const &` the approx because the approx call operator isn't const (not sure why...).

Reply:
:+1: